src/pages-and-resources/discussions/app-list/AppCard.test.jsx

```diff
@@ -1,17 +1,84 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { render, screen } from '@testing-library/react';
+import { Factory } from 'rosie'; // eslint-disable-line import/no-extraneous-dependencies
+
+import AppCard from './AppCard';
+import messages from './messages';
+import '../data/__factories__/index';
+
 describe('AppCard', () => {
+  const partailSupportApp = Factory.build('app');
+  const fullSupportApp = Factory.build('app', { hasFullSupport: true });
+  const selected = true;
+
   test('checkbox input is checked when AppCard is selected', () => {
+    render(
+      <IntlProvider locale="en">
+        <AppCard
+          app={fullSupportApp}
+          onClick={() => jest.fn()}
+          selected={selected}
+        />
+      </IntlProvider>,
+    );
 
+    const radioInput = screen.getByRole('radio');
```

Comment:
Can we use querySelector instead of screen here? e.g
`const { container } = render(render something here)`
`container.querySelector('h3')`

Reply:
Yes we can use but using screen in most preferable in best practices.

Reply:
@awais-ansari Why is screen preferable?